feature/search/src/main/kotlin/uk/govuk/app/search/ui/SearchScreen.kt

```diff
@@ -151,6 +154,9 @@ fun ShowResults(searchResults: List<Result>) {
                         .fillMaxWidth()
                         .clickable(
                             onClick = {
+                                scope.launch {
```

Comment:
The visited item functionality could be moved into the function that handles the search result click.

Reply:
Fixed up